js/meetup/data.js

```diff
@@ -1,4 +1,76 @@
 const meetups = [
+  {
+    episode: 14,
+    title: 'RoR Himalayan Meetup at Kathmandu',
+    date: '2019-09-29',
+    start_time: '11:00 am',
+    end_time: '02:00 pm',
+    address: 'Sainbu, Bhaisepati · Kathmandu',
+    hosted_by: {
+      name: 'Susan Joshi',
+      twitter_username: 'josisusan'
+    },
+    venue_sponsor: {
+      link: 'https://www.cloudfactory.com/',
+      logo: 'https://www.cloudfactory.com/hs-fs/hub/351374/file-461075169-png/img/cf-logo-preview.png?t=1528352709359',
+      name: 'CloudFactory'
+    },
+    sessions: [
+      {
+        id: 1,
+        title: 'Build your free personal Developer website with Jekyll',
+        description: null,
+        speaker: 'Budha Ram Gurung',
+        twitter_username: 'coolbrg',
+        github_username: 'coolbrg',
+        designation: 'CTO, Siris Youth Foundation',
+        slides_link: '',
```

Comment:
https://docs.google.com/presentation/d/1khnzL-VyH0CN_GJaZVSYrq-Dve5Tsbzv9Tsphu41adU/edit?usp=drivesdk

Reply:
The link ask for permission from the author. This is not a public url.

Reply:
@coolbrg could you make the link public?